lib/tasks/practice_editors.rake

```diff
@@ -0,0 +1,16 @@
+namespace :practice_editors do
+  desc 'Adds practice owner to the list of practice editors for their practice'
+  task add_practice_owners_to_practice_editors: :environment do
+    practices = Practice.all
+
+    practices.each do |practice|
+      if practice.user.present? && practice.practice_editors.where(user: practice.user).empty?
+        PracticeEditor.create!(practice: practice, user: practice.user, email: practice.user.email)
```

Comment:
we don't need to set `email` here either now

Reply:
I think we still need the `email` param here, because we need to make sure they have a valid va.gov address before we create a practice editor. Does that make sense?